### DISCUSSION TITLE
outer_boundary_id  in PeripheralTriangleMeshGenerator

### DISCUSSION THREAD

**Discussion by ghost** [idaholab/moose#20582] [Q&A Meshing]
I want to use PeripheralTriangleMeshGenerator to generate a triangulated mesh between a generated outer circle boundary and a provided inner mesh.  The outer_boundary_id is set to define the mesh outer boundary. Then I use FancyExtruderGenerator to extrude the 2D mesh into 3D. But when I define  boundary condition for the radial boundary in my simulation, an error occures saying that this boundary ID does not exist. 
Not sure why this is the case?

**Comment by GiudGiud** [comment]
Hello

Add `show_info = true` to each mesh generator blocks to see if the boundary wasnt defined well in the first place or if it got removed by extrusion. 
You can use another mesh generator such as https://mooseframework.inl.gov/source/meshgenerators/SideSetsAroundSubdomainGenerator.html with a non-fixed normal to paint a radial boundary around a mesh. 

Guillaume

**Reply by ghost** [reply]
Thank you. According to your suggestion, I added show_info= true and found that the ID on the outer boundary was defined with a fixed value of 10000. I can use this ID in my simulation.
